tools/tgsgapcloser/macros.xml

```diff
@@ -0,0 +1,33 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.0.3</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">tgsgapcloser</requirement>
```

Comment:
Indentation...

Reply:
Sorry, I cannot see the problem in that case, could you provide me more details?

Reply:
I saw it, sorry!